RecoTracker/IterativeTracking/python/DetachedTripletStep_cff.py

```diff
@@ -232,7 +232,15 @@
 detachedTripletStep = ClassifierMerger.clone()
 detachedTripletStep.inputClassifiers=['detachedTripletStepClassifier1','detachedTripletStepClassifier2']
 
-trackingPhase1.toReplaceWith(detachedTripletStep, detachedTripletStepClassifier1.clone(
+#LWTNN selector
+from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
+from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
+trackingPhase1.toReplaceWith(detachedTripletStep, TrackLwtnnClassifier.clone(
+     src = 'detachedTripletStepTracks',
+     qualityCuts = [-0.4,0.45,0.9],
+))
+
+trackingPhase1QuadProp.toReplaceWith(detachedTripletStep, detachedTripletStepClassifier1.clone(
```

Comment:
@hajohajo Something went wrong in your rebase. In the recent IB's all occurrances of `trackingPhase1QuadProp` should be removed.